include/container/sycl_iterator.h

```diff
@@ -91,24 +91,24 @@ class BufferIterator {
    * @tparam acc_md_t memory access mode
    * @tparam scalar_t the element type of the buffer
    * @param buff_iterator BufferIterator
-   * @param cgh cl::sycl::handler
+   * @param cgh sycl::handler
    * @param size the region needed to be copied
    */
   template <
-      cl::sycl::access::mode acc_md_t = cl::sycl::access::mode::read_write>
```

Comment:
All instances of `access::mode` can be replaced with `sycl::access_mode` I think - the spec says `access::mode` is deprecated.

Reply:
Thank you for pointing it out to me @hjabird. I addressed all these changes in 1642b50